vnext/ReactUWP/Modules/AppearanceModule.h

```diff
@@ -17,13 +17,14 @@ class AppearanceChangeListener final : public Mso::ActiveObject<> {
   using UISettings = winrt::Windows::UI::ViewManagement::UISettings;
 
  public:
-  AppearanceChangeListener(std::weak_ptr<IReactInstance> &&reactInstance) noexcept;
+  AppearanceChangeListener(std::weak_ptr<IReactInstance> &&reactInstance, Mso::DispatchQueue const &uiQueue) noexcept;
   const char *GetColorScheme() const noexcept;
 
  private:
   static const char *ToString(ApplicationTheme theme) noexcept;
   void OnColorValuesChanged() noexcept;
 
+ private:
```

Comment:
>private [](start = 1, length = 7)

Nit: Why the repetition of the 'private:' marker?

Reply:
Could we leave this as is? I prefer the way I originally wrote this, and don't see why we should add our preferences to other code.

Reply:
Removed it.

---
In reply to: [424025014](https://github.com/microsoft/react-native-windows/pull/4877#discussion_r424025014) [](ancestors = 424025014)